src/JBrowse/View/Dialog/OpenDirectory.js

```diff
@@ -62,7 +62,7 @@ return declare( ActionBarDialog,
     },
 
     show: function( callback ) {
-        dojo.addClass( this.domNode, 'jbrowse fileDialog' );
+        dojo.addClass( this.domNode, 'fileDialog' );
 
```

Comment:
you sure about this change?  The `jbrowse` class was added there for a reason I think.

Reply:
This is a similar fix that was done for the"view details" dialog, I didn't fully investigate but the symptom was just too wide of dialogs like the view details had

Reply:
Eh OK, this dialog is only used in the electron app anyway.